requirements/docs-requirements.txt

```diff
@@ -7,17 +7,16 @@
 git+git://github.com/dimagi/architect@v0.5.7a2#egg=architect==0.5.7a2
 alabaster==0.7.11         # via sphinx
 alembic==0.8.6
-amqp==1.4.9
-anyjson==0.3.3
+amqp==2.3.2
 appdirs==1.4.3
 asn1crypto==0.24.0
 attrs==18.1.0
 babel==2.6.0
-billiard==3.3.0.23
+billiard==3.5.0.4
```

Comment:
I'm confused why this doesn't have a comment about what package it was installed by. requirements.txt seems to have it.

Reply:
Unclear to me too.  Almost none of the requirements in docs-requirements.txt note the package that requires them.  In any case it doesn't look specific to the changes here.